products/slackware.md

```diff
@@ -0,0 +1,109 @@
+---
+permalink: /slackware
+alternate_urls:
+-   /slackwarelinux
+title: Slackware Linux
+category: os
+releasePolicyLink: http://www.slackware.com/changelog/
+iconSlug: slackware
+changelogTemplate: http://mirrors.slackware.com/slackware/slackware-__RELEASE_CYCLE__/ChangeLog.txt
+activeSupportColumn: false
+versionCommand: cat /etc/os-release
+releaseColumn: false
+releaseDateColumn: true
+realseLabel: "__RELEASE_CYCLE__ (__CODENAME__)"
+releasePolicyLink: http://www.slackware.com/faq/do_faq.php?faq=general#4
+purls:
+-   purl: pkg:os/slackwarelinux
+auto:
+-   distrowatch: slackware
+    regex: '^Distribution Release: Slackware Linux (?P<major>\d)\.(?P<minor>\d)$'
```

Comment:
```suggestion
    regex: '^Distribution Release: Slackware (Linux )?(?P<major>\d+)\.(?P<minor>\d+)$'
```

https://regex101.com/r/Q5uFOI/1

Double digits, and older release didn't use `Linux` in the title